LoopKit/Notification/LoopNotificationCategory.swift

```diff
@@ -18,4 +18,8 @@ public enum LoopNotificationCategory: String {
     case pumpExpired
     case pumpFault
     case alert
```

Comment:
Added these for the local notifications that will present for remote bolus/carbs.

Reply:
These alert definitions are probably going away, or being moved into Loop itself. But I'll take this for now